app/assets/stylesheets/puppetclasses.scss

```diff
@@ -32,8 +32,12 @@
   .matcher .matcher-group {
     display: block;
   }
-  
+
   .matcher_value {
     width: 60%;
   }
 }
+
+.selected_puppetclass.unavailable a {
+  color: #d1d1d1;
```

Comment:
https://github.com/theforeman/foreman/pull/4048#discussion_r91099151 - should be a patternfly color